### DISCUSSION TITLE
Support `required` prop for date range selection

### DISCUSSION THREAD

**Discussion by homerchen19** [gpbl/react-day-picker#1736] [Ideas]
The `required` prop is currently only supported for single date selection. I am wondering if it would be possible to extend this support to date range selection as well. This would ensure that users are unable to unselect the range start date. One example of this functionality can be found in [MUI's DateRangePicker](https://mui.com/x/react-date-pickers/date-range-picker/#basic-usage). Once a start date is selected, regardless of any further interactions with the calendar, the range start date always remains selected.

**Comment by gpbl** [comment]
Thanks for your suggestion, we will work on this in #1835.
